eng/Version.Details.xml

```diff
@@ -5,9 +5,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>15f00efd583eab4372b2e9ca25bd80ace5b119ad</Sha>
     </Dependency>
-    <Dependency Name="Private.SourceBuild.ReferencePackages" Version="1.0.0-beta.20107.1">
+    <Dependency Name="Private.SourceBuild.ReferencePackages" Version="1.0.0-beta.20220.1">
       <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
-      <Sha>9cc7dad6c4d9302c00803df5dea41c8a7a59e047</Sha>
+      <Sha>f3db8b2a4a7c12ac44283b516dac8cda12a5edeb</Sha>
```

Comment:
This doesn't match the version down below, should it?

https://github.com/dotnet/source-build/blob/087941b9db4d583439643e8d06a6bb27afd46dd5/eng/Versions.props#L10

Reply:
It doesn't especially matter since SBRP isn't ubercloned, but fixed for clarity.